slicer-plugin/NvidiaAIAA/SegmentEditorNvidiaAIAALib/SegmentEditorEffect.py

```diff
@@ -0,0 +1,611 @@
+import atexit
+import json
+import logging
+import os
+import shutil
+import tempfile
+import time
+
+import SimpleITK as sitk
+import qt
+import sitkUtils
+import slicer
+import vtk
+from SegmentEditorEffects import *
+
+from AIAAClient import AIAAClient
+
+
+class SegmentEditorEffect(AbstractScriptedSegmentEditorEffect):
+    """This effect uses Watershed algorithm to partition the input volume"""
+
+    def __init__(self, scriptedEffect):
+        scriptedEffect.name = 'Nvidia AIAA'
```

Comment:
should this name matched the name we have upthere for testing?

Reply:
It's part of slicer plugin template